modules/core/src/weaver/suites.scala

```diff
@@ -91,12 +91,13 @@ trait MutableFSuite[F[_]] extends ConcurrentEffectSuite[F]  {
   def pureTest(name: String)(run : => Expectations) :  Unit = registerTest(name)(_ => Test(name, effect.delay(run)))
   def simpleTest(name:  String)(run: => F[Expectations]) : Unit = registerTest(name)(_ => Test(name, effect.suspend(run)))
   def loggedTest(name: String)(run: Log[F] => F[Expectations]) : Unit = registerTest(name)(_ => Test(name, log => run(log)))
-  def test(name: String) : PartiallyAppliedTest = new PartiallyAppliedTest(name)
+  def test(name: String) : PartiallyAppliedTest = new PartiallyAppliedTest { val testName: String = name }
```

Comment:
I don't think the change is absolutely necessary here, in the monix one you can do 

``` 
def test(name: String) : PartiallyAppliedTest = new this.PartiallyAppliedTest(name) 
class PartiallyAppliedTest(name) extends super.PartiallyAppliedTest(name) 
```

Reply:
Addressed in c82f37f572c601d2209501fded780c7abee49130. I had to rename the subclass because of a compiler warning that this kind of shadowing is deprecated.